apps/worker/src/interfaces/batch-announcer.job.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface IBatchAnnouncerJobData {
```

Comment:
shouldn't this have a list of all the announcements in the batch?

Reply:
I intend to do that when logic is being put in place